Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -89,7 +165,22 @@ public static int[] InsertLast(int[] array, int number)
         public static int[] InsertAt(int[] array, int number, int index)
         {
             // ToDo: implement.
-            return array;
+            if (array != null)
+            {
+                int[] newArr = new int[array.Length + 1];
+                newArr[index] = number;
+                for (int i = 0; i < array.Length; ++i)
+                    if (i < index)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)